src/scales/categoryScale.ts

```diff
@@ -11,10 +11,9 @@ export module Scale {
     public _typeCoercer: (d: any) => any = (d: any) => d != null && d.toString ? d.toString() : d;
 
     /**
-     * Creates an OrdinalScale.
+     * Creates a CategoryScale.
      *
-     * An OrdinalScale maps strings to numbers. A common use is to map the
-     * labels of a bar plot (strings) to their pixel locations (numbers).
+     * A CategoryScale maps domain values to pixel ranges.
```

Comment:
"maps strings to numbers" was a more accurate description, since that's literally what the scale does (see the generic declaration in the class line).